guides/common/modules/proc_using-vmware-cloud-init-and-userdata-templates-for-provisioning.adoc

```diff
@@ -10,7 +10,7 @@ ifdef::satellite[]
 image::common/user-data-sequence-satellite.png[]
```

Comment:
Please add alt texts line 10 and 16.

Reply:
Already done - https://github.com/theforeman/foreman-documentation/pull/2843